porchlight/door.py

```diff
@@ -687,9 +689,7 @@ def map_arguments(self):
 
                 # Change keyword arguments as well.
                 if old_name in self.keyword_args:
-                    self.keyword_args[mapped_name] = self.keyword_args[
-                        old_name
```

Comment:
This is formatted pretty roughly. I would be good to have this be on one line, maybe assign `self.keyword_args` to a temporary variable.

Reply:
lol I'm dumb